examples/pytorch/CaptumExample/Titanic_Captum_Interpret.py

```diff
@@ -0,0 +1,323 @@
+    summary, _ = count_model_parameters(net)
+    mlflow.log_text(str(summary), "model_summary.txt")
+    return (
+        net,
+        train_features,
+        train_labels,
+        test_features,
+        test_labels,
+        feature_names,
+    )
+
+
+def train_step(train_features):
+    train_input_tensor = torch.from_numpy(train_features).type(torch.FloatTensor)
+    out_probs = net(train_input_tensor).detach().numpy()
+    out_classes = np.argmax(out_probs, axis=1)
+    mlflow.log_metric("Train Accuracy", float(sum(out_classes == train_labels) / len(train_labels)))
+    print("Train Accuracy:", sum(out_classes == train_labels) / len(train_labels))
+    return train_input_tensor
+
+
+def test_step(test_features):
+    test_input_tensor = torch.from_numpy(test_features).type(torch.FloatTensor)
+    out_probs = net(test_input_tensor).detach().numpy()
+    out_classes = np.argmax(out_probs, axis=1)
+    mlflow.log_metric("Test Accuracy", float(sum(out_classes == test_labels) / len(test_labels)))
+    print("Test Accuracy:", sum(out_classes == test_labels) / len(test_labels))
+    return test_input_tensor
+
+
+def feature_conductance(test_input_tensor):
+    ig = IntegratedGradients(net)
+    test_input_tensor.requires_grad_()
+    attr, _ = ig.attribute(test_input_tensor, target=1, return_convergence_delta=True)
+    attr = attr.detach().numpy()
+    # To understand these attributions, we can first average them across all the inputs and print and visualize the average attribution for each feature.
+    feature_imp, feature_imp_dict = visualize_importances(feature_names, np.mean(attr, axis=0))
+    mlflow.log_metrics(feature_imp_dict)
+    mlflow.log_text(str(feature_imp), "feature_imp_summary.txt")
+    fig, (ax1, ax2) = plt.subplots(2, 1)
+    fig.tight_layout(pad=3)
+    ax1.hist(attr[:, 1], 100)
+    ax1.set(title="Distribution of Sibsp Attribution Values")
+
+    # we can bucket the examples by the value of the sibsp feature and plot the average attribution for the feature.
+    # In the plot below, the size of the dot is proportional to the number of examples with that value.
+
+    bin_means, bin_edges, _ = stats.binned_statistic(
+        test_features[:, 1], attr[:, 1], statistic="mean", bins=6
+    )
+    bin_count, _, _ = stats.binned_statistic(
+        test_features[:, 1], attr[:, 1], statistic="count", bins=6
+    )
+
+    bin_width = bin_edges[1] - bin_edges[0]
+    bin_centers = bin_edges[1:] - bin_width / 2
+    ax2.scatter(bin_centers, bin_means, s=bin_count)
+    ax2.set(xlabel="Average Sibsp Feature Value", ylabel="Average Attribution")
+    mlflow.log_figure(fig, "Average_Sibsp_Feature_Value.png")
+
+
+def layer_conductance(test_input_tensor):
+    """
+    To use Layer Conductance, we create a LayerConductance object passing in the model as well as the module (layer) whose output we would like to understand. In this case, we choose net.sigmoid1, the output of the first hidden layer.
```

Comment:
Is there any documentation on Layer Conductance that we could link to, in case the reader doesn't know what it is (e.g. I have no idea what it is :P)

Reply:
Maybe we could provide such links in the README, before the reader gets to looking at this code

Reply:
@smurching Sure. I am gonna add the required documentation links in the README  :-)

Reply:
@Arvind-Ideas2IT Can we wrap the docstring?

Reply:
@harupy  Sure,Mate.